Account/FxAState.swift

```diff
@@ -156,21 +156,34 @@ public class SeparatedState: FxAState {
 }
 
 // Not an externally facing state!
-public class ReadyForKeys: FxAState {
+public class FxASessionState: FxAState {
```

Comment:
We have `TokenAndKeys`.  Perhaps `TokenState`?  I don't care too much, but the `FxA` prefix is verbose.